verso/src/lib.rs

```diff
@@ -174,6 +176,7 @@ impl VersoviewController {
     }
 
     /// Create a new verso instance with custom settings and get the controller to it
+    #[deprecated = "Use VersoBuilder instead"]
```

Comment:
I think we can remove `new_with_settings` directly since it's in a very early stage. A builder pattern is much better